react/features/mobile/call-integration/middleware.ts

```diff
@@ -272,10 +277,14 @@ function _conferenceWillJoin({ dispatch, getState }: IStore, next: Function, act
         return result;
     }
 
-    // When assigning the call UUID, do so in upper case, since iOS will return
+    // When assigning the callUUID, do so in upper case, since iOS will return
     // it upper-cased.
     conference.callUUID = (callUUID || uuidv4()).toUpperCase();
 
+    if (callUUID) {
```

Comment:
You should be able to drop this now. The reason why we didn't find the call earlier is because the CallKit provider was reset.

Reply:
Addressed.